File: Examples/Conversations/Obj-C/ConversationsExample/MyCommentTableViewCell.m
//
//  MyCommentTableViewCell.m
//  ConversationsExample
//
//  Copyright © 2017 Bazaarvoice. All rights reserved.
//

#import "MyCommentTableViewCell.h"

@interface MyCommentTableViewCell ()

@property (weak, nonatomic) IBOutlet UILabel *commentTitle;
@property (weak, nonatomic) IBOutlet UILabel *commentText;


@end


@implementation MyCommentTableViewCell

@synthesize comment = _comment;

- (void)setComment:(BVComment *)comment{
    
    _comment = comment;
    self.commentTitle.text = self.comment.title ? self.comment.title : @"No title";
    self.commentText.text =  self.comment.commentText ? self.comment.commentText : @"No comment text";
    
}



@end

File: Examples/Conversations/Swift/ConversationsExample/MyCommentTableViewCell.swift
//
//  MyCommentTableViewCell.swift
//  ConversationsExample
//
//  Copyright © 2017 Bazaarvoice. All rights reserved.
//

import UIKit
import BVSDK

class MyCommentTableViewCell: UITableViewCell {
  @IBOutlet var commentTitle: UILabel!
  @IBOutlet var commentLabel: UILabel!
  
  var comment : BVComment? {
    didSet {
      commentTitle.text = comment?.title ?? ""
      commentLabel.text = comment?.commentText ?? "Error"
    }
  }
}
